juju/application.py

```diff
@@ -109,6 +109,18 @@ def tag(self):
     async def add_relation(self, local_relation, remote_relation):
         """Add a relation to another application.
 
+        DEPRCATED: Use `relate` instead
```

Comment:
typo: DEPRECATED

Reply:
Good spot, fixed

I have also change formatting to be consistent with the rest of pylibjuju